shell/src/test/java/tachyon/shell/TFsShellUtilsTest.java

```diff
@@ -42,4 +69,175 @@ public void getFilePathTest() throws IOException {
+    mTfs.mkdir(new TachyonURI("/testWildCards/foo"));
+    mTfs.mkdir(new TachyonURI("/testWildCards/bar"));
+    fileIds[0] = 
+        TachyonFSTestUtils.createByteFile(mTfs, "/testWildCards/foo/foobar1", 
+            WriteType.MUST_CACHE, 10);
+    fileIds[1] =
+        TachyonFSTestUtils.createByteFile(mTfs, "/testWildCards/foo/foobar2", 
+            WriteType.MUST_CACHE, 20);
+    fileIds[2] =
+        TachyonFSTestUtils.createByteFile(mTfs, "/testWildCards/bar/foobar3", 
+            WriteType.MUST_CACHE, 30);
+    fileIds[3] =
+        TachyonFSTestUtils.createByteFile(mTfs, "/testWildCards/foobar4", 
+            WriteType.MUST_CACHE, 40);
+    return fileIds;
+  }
+  
+  @Test
+  public void getTachyonURIsTest() throws IOException {
+    resetTachyonFileHierarchy();   
+    String rootDir = "/testWildCards";
+    
+    List<TachyonURI> tl1 = TFsShellUtils.getTachyonURIs(mTfs, new TachyonURI(rootDir + "/foo"));
+    Collections.sort(tl1, createTachyonURIComparator());
+    Assert.assertEquals(tl1.size(), 1);
+    Assert.assertEquals(tl1.get(0).getPath(), rootDir + "/foo");
+
+    // Trailing slash
+    List<TachyonURI> tl2 = TFsShellUtils.getTachyonURIs(mTfs, new TachyonURI(rootDir + "/foo/"));
+    Collections.sort(tl2, createTachyonURIComparator());
+    Assert.assertEquals(tl2.size(), 1);
+    Assert.assertEquals(tl2.get(0).getPath(), rootDir + "/foo");
+    
+    // Wildcard
+    List<TachyonURI> tl3 = TFsShellUtils.getTachyonURIs(mTfs, new TachyonURI(rootDir + "/foo/*"));
+    Collections.sort(tl3, createTachyonURIComparator());
+    Assert.assertEquals(tl3.size(), 2);
+    Assert.assertEquals(tl3.get(0).getPath(), rootDir + "/foo/foobar1");
+    Assert.assertEquals(tl3.get(1).getPath(), rootDir + "/foo/foobar2");
+
+    // Trailing slash + wildcard
+    List<TachyonURI> tl4 = TFsShellUtils.getTachyonURIs(mTfs, new TachyonURI(rootDir + "/foo/*/"));
+    Collections.sort(tl4, createTachyonURIComparator());
+    Assert.assertEquals(tl4.size(), 2);
+    Assert.assertEquals(tl4.get(0).getPath(), rootDir + "/foo/foobar1");
+    Assert.assertEquals(tl4.get(1).getPath(), rootDir + "/foo/foobar2");
+
+    // Multiple wildcards
+    List<TachyonURI> tl5 = TFsShellUtils.getTachyonURIs(mTfs, new TachyonURI(rootDir + "/*/foo*"));
+    Collections.sort(tl5, createTachyonURIComparator());
+    Assert.assertEquals(tl5.size(), 3);
+    Assert.assertEquals(tl5.get(0).getPath(), rootDir + "/bar/foobar3");
+    Assert.assertEquals(tl5.get(1).getPath(), rootDir + "/foo/foobar1");
+    Assert.assertEquals(tl5.get(2).getPath(), rootDir + "/foo/foobar2");
+  }
+  
+  public void resetLocalFileHierarchy() throws IOException {
+    /**
+     * Generate such local structure
+     *  /testWildCards
+     *  ├── foo
+     *  |    ├── foobar1
+     *  |    └── foobar2
+     *  ├── bar
+     *  |    └── foobar3
+     *  └── foobar4
+     */
+    FileUtils.deleteDirectory(new File(mLocalTachyonCluster.getTachyonHome() + "/testWildCards"));
```

Comment:
`TemporaryFolder` can be used to create a temporary folder here, the folder will be automatically deleted from local filesystem after the unit test, example is here: https://github.com/amplab/tachyon/blob/master/servers/src/test/java/tachyon/worker/block/evictor/EvictorTestBase.java#L47